ui/app/components/secret-edit.js

```diff
@@ -163,7 +164,7 @@ export default Component.extend(FocusOnInsertMixin, {
   }),
 
   transitionToRoute() {
-    this.router.transitionTo(...arguments);
+    return this.router.transitionTo(...arguments);
```

Comment:
What does returning here do? Still learning Ember

Reply:
The EC task needs to yield the promise in order to function properly and here were weren't returning it. The controller `transitionToRoute` already does return the Transition, which is a promise so this makes it so they match.